composeApp/src/commonMain/kotlin/com/github/familyvault/services/IChatService.kt

```diff
@@ -12,11 +13,14 @@ interface IChatService {
     fun sendTextMessage(chatThreadId: String, messageContent: String, respondToMessageId: String)
     fun sendVoiceMessage(chatThreadId: String, audioData: ByteArray)
     fun getVoiceMessage(fileId: String) : ByteArray
-    suspend fun createGroupChat(name: String, members: List<FamilyMember>): ChatThread
-    suspend fun updateChatThread(thread: ChatThread, members: List<FamilyMember>, newName: String?)
+    suspend fun createGroupChat(name: String, members: List<FamilyMember>, currentUser: FamilyMember): ChatThread
+    suspend fun updateChatThread(thread: ChatThread, members: List<FamilyMember>, newName: String?, currentUser: FamilyMember? = null)
```

Comment:
Czym jest `currentUser`? Jeżeli jest to twórca grupy chatu to może nazwać go lepiej np. `groupChatCreator`